<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newTitle</class>
 <widget class="QWidget" name="newTitle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add title</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0" colspan="4">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="textLabel1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Video file</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KUrlRequester" name="urFile"/>
     </item>
     <item>
      <widget class="QLabel" name="lTotalTime">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>--:--:--</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="3">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>31</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>341</width>
       <height>28</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="KPushButton" name="bAdd">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
     <property name="shortcut">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="buttonGroup1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbNumber">
        <property name="text">
         <string>number of chapters</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="rbLength">
        <property name="text">
         <string>chapter length</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTimeEdit" name="twLength">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="displayFormat">
         <string>HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>131</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="ckReencode">
        <property name="text">
         <string>Do not reencode</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="nbChapters">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>99</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">kpushbutton.h</include>
  <include location="local">kurlrequester.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>urFile</sender>
   <signal>urlSelected(const KUrl&amp;)</signal>
   <receiver>newTitle</receiver>
   <slot>urlSelected(const KUrl&amp;)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bAdd</sender>
   <signal>clicked()</signal>
   <receiver>newTitle</receiver>
   <slot>bAddClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbNumber</sender>
   <signal>clicked()</signal>
   <receiver>newTitle</receiver>
   <slot>rbNumberClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbLength</sender>
   <signal>clicked()</signal>
   <receiver>newTitle</receiver>
   <slot>rbLengthClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>rbNumberClicked()</slot>
  <slot>rbLengthClicked()</slot>
 </slots>
</ui>
